### DISCUSSION TITLE
Its Fine using dynamic import on /component level? not /pages level?

### DISCUSSION THREAD

**Discussion by isggwp** [vercel/next.js#18290] [Help] [answered]
Its Fine using dynamic import on component level? not pages level? i just think about minimum repeat code in another pages.

**Comment by jamesmosier** [comment, marked as answer]
You can use dynamic imports to dynamically import components into other components or dynamically import component into pages. Dynamic imports do not work for importing pages though 😄

**Comment by isggwp** [comment]
ok thanks @jamesmosier :smile:
